src/layers/ZoneLayer/ZoneLayer.js

```diff
@@ -48,14 +49,28 @@ class ZoneLayer extends CasaLayer {
   constructor(options = {}) {
     super({
       name: 'Verbundzonen',
-      olLayer: new OLVectorLayer({
-        className: 'Verbundzonen', // needed for forEachLayerAtPixel
-        source: new VectorSource(),
-        style: (f, r) => this.zoneStyle(f, r),
+      olLayer: new LayerGroup({
+        layers: [
+          new OLVectorLayer({
+            className: 'Verbundzonen', // needed for forEachLayerAtPixel
```

Comment:
What's so important with the className ? you never use it.

Reply:
This was in the code before, I just copied it for the layers. It is actually important in the olLayer (removing will break the app), but in the layers I could probably remove it